components/ProviderLogo.tsx

```diff
@@ -1,10 +1,11 @@
 import Image from "next/image";
 import type { RepoProvider } from "../utils/providerAPI";
 import React from "react";
+import { Theme } from "../utils/theme";
 
-export const getProviderLogoSrc = (provider: RepoProvider, theme: "light" | "dark") => `/${provider}${theme === 'dark' ? "-dark" : ""}.svg`;
+export const getProviderLogoSrc = (provider: RepoProvider, theme: Theme) => `/${provider}${theme !== 'dark' ? "-dark" : ""}.svg`;
```

Comment:
is this correct? why prefixing `-dark` when `!== 'dark'`

Reply:
Yes, this is correct. The darker version of the logo should be shown when the theme is light. The lighter version of the logo should be shown when the theme is dark. I know it can be confusing, but it was confusing when we had things the other way around as well 😅